eval/action.yml

```diff
@@ -0,0 +1,66 @@
+name: "Sandard Eval"
+description: "Evaluate Standard based projects"
+
+inputs:
+  key_path:
+    description: "The path to place the Nix secret Key"
+    required: true
+    default: "${{ runner.temp }}/.nix-key"
+  github_pat:
+    description: "GitHub Token for Accessing Private Repos with Nix"
+    required: true
+    default: ${{ github.token }}
+  nix_key:
+    description: "Secret Key used for Nix Package Signing"
+    required: false
+  s3_key:
+    description: "Key to upload to S3 Buckets with Nix"
+    required: false
+  s3_id:
+    description: "ID to upload to S3 Buckets with Nix"
+    required: false
+  cache:
+    description: "Nix cache to optionally upload artifacts to"
+    # auto refers to the local nix store, so nothing is uploaded by default
+    default: auto
+    required: false
+
+
+runs:
+  using: "composite"
+  steps:
+    - name: Emplace Secrets
+      run: ${{ github.action_path }}/secrets.sh
+      shell: bash
+      env:
+        NIX_SECRET_KEY: ${{ inputs.nix_key }}
+        NIX_KEY_PATH: ${{ inputs.key_path }}
+        AWS_ACCESS_KEY_ID: ${{ inputs.s3_id }}
+        AWS_SECRET_ACCESS_KEY: ${{ inputs.s3_key }}
+
+    - name: Install Nix
+      uses: cachix/install-nix-action@v18
+      with:
+        extra_nix_config: |
+          accept-flake-config = true
+          trusted-users = runner
+          access-tokens = github.com=${{ inputs.github_pat }}
+          secret-key-files = ${{ inputs.key_path }}
+
+    - name: Run Eval
```

Comment:
```suggestion
    - name: Discover build & run targets
```

Reply:
we aren't running targets in this phase though

Reply:
Ah yeah! The intent was "discover build targets & discover run targets". Hahaha

Reply:
ah yes I guess it could be read both ways